files/angular-material-icons/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "angular-material-icons",
+  "repo": "klarsys/angular-material-icons",
+  "files": {
+    "include": ["angular-material-icons.js"]
```

Comment:
Oh yes. My bad. I created minified version after writing update.json.

I can fix it right now and generate  anew pull request.

On Fri, Jan 30, 2015 at 9:30 PM, Graeme Yeates <notifications@github.com>
wrote:

> In files/angular-material-icons/update.json
> <https://github.com/jsdelivr/jsdelivr/pull/3257#discussion_r23850919>:
>
> > @@ -0,0 +1,8 @@
> > +{
> > +  "packageManager": "bower",
> > +  "name": "angular-material-icons",
> > +  "repo": "klarsys/angular-material-icons",
> > +  "files": {
> > +    "include": ["angular-material-icons.js"]
>
> Did you not mean to include this file?
>
> https://github.com/klarsys/angular-material-icons/blob/master/angular-material-icons.min.js
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsdelivr/jsdelivr/pull/3257/files#r23850919>.
>